src/transformers/modeling_utils.py

```diff
@@ -3182,7 +3221,12 @@ def from_pretrained(
         keep_in_fp32_modules = None
         use_keep_in_fp32_modules = False
 
-        if pretrained_model_name_or_path is not None:
+        if from_gguf is not None and hf_quantizer is not None:
+            raise ValueError(
+                "You cannot combine Quantization and loading a model from a GGUF file, try again by making sure you did not passed a `quantization_config` or that you did not loaded a quantized model from the Hub."
+            )
```

Comment:
Once the file is loaded in a pt state dict, quantization cannot be applied?

Reply:
(happy to not support that for now, indeed)

Reply:
Might be complicated as we would only support the quant schemes that do not require data calibration and would require many patches and if/else checks everywhere :/

Reply:
berk